packages/flutter/test/material/date_picker_theme_test.dart

```diff
@@ -517,6 +531,8 @@ void main() {
         'cancelButtonStyle: ButtonStyle#00000(foregroundColor: WidgetStatePropertyAll(${const Color(0xffffff6f)}))',
         'confirmButtonStyle: ButtonStyle#00000(foregroundColor: WidgetStatePropertyAll(${const Color(0xffffff7f)}))',
         'locale: en',
+        'subheaderTextStyle: TextStyle(inherit: true, size: 13.0)',
+        'headerNavigationButtonColor: ${const Color(0xffffff9f)}',
```

Comment:
Should the unit test below "DatePickerDialog uses ThemeData datePicker theme (calendar mode)" also be updated and verify whether the subheaders are correctly customized? Either updating the existing tests or adding a new unit test to verify the new properties are able to customize the subheaders is fine:)